apps/web/src/components/r3f/scenes/pack-opening/sections/particles.tsx

```diff
@@ -98,6 +98,7 @@ export default function Particles({
         args={[new BufferGeometry(), new Material(), count]}
       >
         <sphereBufferGeometry attach="geometry" />
+        {/* @ts-ignore ignore "Type instantiation is excessively deep and possibly infinite." */}
```

Comment:
Annoying, but I don't see another way around this for now.

Reply:
Interesting. I'll take a close look Monday and see if anything obvious comes to mind.